components/MaskedTransition/src/private/MDCMaskedPresentationController.h

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@protocol MDMTransitionContext;
+
+FOUNDATION_EXPORT UIView *CreateScrimView(id<MDMTransitionContext> context);
```

Comment:
If the presentationMode is not custom, I would not expect custom presentation elements.
My understanding is the UIPresentationController (iOS 8) was created to overcome shortcoming in the transitioningDelegate system (iOS 7).
I don't think that the added complexity of exposing this scrim through CreateScrimView is a good idea, especially as the scrimView is owned by either the context or the _presentationLayer.  I would expect the owner to control the add/remove and alpha of any child views.